prow/jobs/busola/busola-web/busola-web.yaml

```diff
@@ -13,7 +13,7 @@ presubmits: # runs on PRs
         preset-sa-gcr-push: "true"
       run_if_changed: '^core/|^core-ui/|^service-catalog-ui/|^components/|^shared/|^package.json'
       optional: true
-      skip_report: false
+      skip_report: true
```

Comment:
Do you want these jobs to be not visible on GitHub by default? If you set `skip_report` to `true` then you will disable reporting this job completely. That means even status on GitHub will not be shown.